app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -7,11 +7,9 @@ import type { FormStepComponentProps } from '@18f/identity-form-steps';
 import GeneralError from './general-error';
 import TipList from './tip-list';
 import { SelfieCaptureContext } from '../context';
-import {
-  DocumentCaptureSubheaderOne,
-  SelfieCaptureWithHeader,
-  DocumentFrontAndBackCapture,
-} from './documents-and-selfie-step';
+import { DocumentCaptureSubheaderOne } from './documents-and-selfie-step';
+import { DocumentsCaptureStepComponent } from './documents-step';
```

Comment:
```suggestion
import { DocumentCaptureStep } from './documents-step';
```

we're capturing 1 document ... as a naming convention, it doesn't appear we commonly name our components "...Component"

i think we can either:
1.  revert the renaming to `DocumentFrontAndBackCapture`
2. rename to something like `DocumentCaptureStep` or `DocumentStep`